benchmarks/benchmark_rowwise_scaled_linear_cutlass.py

```diff
@@ -0,0 +1,68 @@
+import pandas as pd
+import torch
+from tqdm import tqdm
+
+from torchao.ops import (
+    rowwise_scaled_linear_cutlass_s4s4,
+    rowwise_scaled_linear_cutlass_s8s4,
+)
+from torchao.utils import benchmark_torch_function_in_microseconds
+
+
+def get_problem(m: int, n: int, k: int, A_nbits: int, B_nbits: int):
+    assert A_nbits in (4, 8) and B_nbits in (4, 8)
+
+    dev = torch.device("cuda")
+    A = torch.randint(-128, 127, (m, k * A_nbits // 8), dtype=torch.int8, device=dev)
+    A_scale = torch.randn((m,), dtype=torch.half, device=dev)
+    B = torch.randint(
+        -128, 127, size=(n, k * B_nbits // 8), dtype=torch.int8, device=dev
+    )
+    B_scale = torch.randn((n,), dtype=torch.half, device=dev)
+    C = None
+
+    return A, A_scale, B, B_scale, C
+
+
+def benchmark(m: int, k: int, n: int):
+    dev = torch.device("cuda")
+    A_ref = torch.randn((m, k), dtype=torch.half, device=dev)
+    B_ref = torch.randn((n, k), dtype=torch.half, device=dev)
+    fp16_time = benchmark_torch_function_in_microseconds(
+        torch.nn.functional.linear, A_ref, B_ref
+    )
```

Comment:
@alexsamardzic For the FP16 timings, I changed from `torch.matmul((m,k), (k,n))` to `F.linear((m,k), (n,k))` so it matches the low-bit linear kernel better. Hope it is fine?

Reply:
Sure, that's fine.